### DISCUSSION TITLE
CLI command for clone and delete

### DISCUSSION THREAD

**Discussion by arbadzukhron** [deta/space-docs#83] [Q&A] [answered]
How to clone project from Deta Builder for development locally? I use `space link` and `space dev` but i don't know where to find the `project ID`. There is no `project ID` in `Develop` tab.

And how to delete project from Deta Builder?

**Comment by sudo-subg** [comment, marked as answer]
Hey @arbadzukhron, as of now, we don't have any way to clone Space projects, but we will have `space clone` to do so in the near future. The same goes for deleting projects; we currently don't have the option to delete projects but it's something we'll have soon.

> where to find the project ID?

Click on the `Builder` app in your Canvas → click on the project you want to link using `space link` → at the bottom of the screen click on the `Teletype` bar → click on the `Copy Project ID` option
![project-id](https://user-images.githubusercontent.com/102468247/232011135-9e6f9111-ba66-4d8b-9c74-49a22c370276.gif)
